build.sbt

```diff
@@ -138,6 +138,7 @@ lazy val packageRpmForLoader = taskKey[File]("Create rpm package for active serv
 lazy val packagingSettings = Seq(
   bashScriptExtraDefines += IO.read((baseDirectory.value / "project" / "NativePackagerSettings" / "extra-defines.bash")),
   mappings in (Compile, packageDoc) := Seq(),
+  isSnapshot in ThisBuild := false,
```

Comment:
I thought at one point we had isSnapshot being inferred by the version pointing to a tag or not. But, I could get behind this. We really don't do snapshot builds anyways.